### ISSUE TITLE
"Motivational words for errors" are being displayed for all errors

### ISSUE THREAD

**Issue by AshlinHarris** [fonsp/Pluto.jl#3062]
#3040 mentions that motivational words wil be displayed at a low frequency, but I'm seeing them for 100% of error messages.

Could you please add a toggle switch to disable all motivational words? My team prepares Pluto notebooks for users who are not programmers or Julia users, so if an error occurs, they could be confused by the humor or the link to the Julia Discourse

**Comment by pankgeorg** [comment]
There is always the low tech solution of adding this cell:

```
html"""<style>.dont-panic{ display: none }</style>"""
```

**Comment by ederag** [comment]
Overall, the new error presentation feels much clearer, thanks !

But here is the story of my encounter with "Keep calm, you got this!":
I was calm, as the errors were expected and easy to fix.
But seeing the computer telling me how to behave ("keep calm"),
and saying something I knew already ("you got this")
did not cheer me up (which was the intent I suppose),
but rather broke my concentration
(where is it coming from, why on some errors only, ... ?)

And after some use I still find those "Motivational words for errors" at best distracting (_not_ entertaining :wink:)
(especially as they appear randomly, changing, and prominent).


By the way, I quickly searched for this issue last week, to no avail,
so to help people searching for the message itself,
here is the list of messages (found in [frontend/components/ErrorMessage.js](https://github.com/fonsp/Pluto.jl/blob/d2dfa8cec3c3d0d6e1c35acd1c4ae6b3fce58c5a/frontend/components/ErrorMessage.js#L549-L566)):

    "Don't panic!",
    "Keep calm, you got this!",
    "You got this!",
    "Silly computer!",
    "Silly computer!",
    "beep boop CRASH 🤖",
    "computer bad, you GREAT!",
    "Probably not your fault!",
    "Try asking on Julia Discourse!",
    "uhmmmmmm??!",
    "Maybe time for a break? ☕️",
    "Everything is going to be okay!",
    "Computers are hard!",
    "C'est la vie !",
    "¯\\_(ツ)_/¯",
    "Oh no! 🙀",
    "this suckz 💣",
    "Be patient :)"

Final note, about the "link to Julia Discourse":
[the few lines below](https://github.com/fonsp/Pluto.jl/blob/d2dfa8cec3c3d0d6e1c35acd1c4ae6b3fce58c5a/frontend/components/ErrorMessage.js#L577-L579) explain why suddenly a discourse page appears sometimes.
But normally when a click might link elsewhere, the link does (or should) appear in status bar. Here it is silent.

**Comment by BioTurboNick** [comment]
Yeah... I'm using Pluto in a professional context, and these are kind of embarrassing. I'm glad I'm still on v0.19 at the moment...

**Comment by pankgeorg** [comment]
> and these are kind of embarrassing

This is not our intent at all. I've been in the business, client facing world and I can relate with the feeling. We are investigating Pluto-themed approaches, I'll keep you posted.

**Comment by joshday** [comment]
I would vastly prefer an `ENV` variable I can set in my `startup.jl` to disable these.

I frankly want to turn them off forever for both myself and customers I'm building for.

**Comment by tamasgal** [comment]
Any updates on this? Recently did a presentation and these messages were neither helpful, nor professional:

<img width="280" alt="Image" src="https://github.com/user-attachments/assets/b2142dcf-f9ad-49aa-8340-33c160867b68" />

**Comment by tamasgal** [comment]
To be honest; I am not so happy how things are going with features/PRs. Maybe it's time for a fork.

**Comment by tamasgal** [comment]
This issue is half a year old and open. Other issues and PRs with features requests/additions were opinionatedly closed by the main dev (there were several in the past). It's not a threat, it's how things develop of people disagree.
